contracts/src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol

```diff
@@ -385,6 +311,78 @@ contract ERC20Bridge is
         return abi.encode(BridgeAction.Transfer, paramsData);
     }
 
+    /**
+     * @dev See {TeleporterUpgradeable-receiveTeleporterMessage}.
+     *
+     * Receives a Teleporter message and routes to the appropriate internal function call.
+     */
+    function _receiveTeleporterMessage(
+        bytes32 originBlockchainID,
+        address originSenderAddress,
```

Comment:
We've renamed these to `origin`, which I think is better than `native`. Can we replace the rest of the variables in this contract to align as well?

Reply:
Looks like we also use `source` in places. Might be a good idea to align what we can, though some places different terms might make more sense.

Reply:
I took a look through `ERC20Bridge` on the different places we use origin/source, and I think the rest of places that use `native` look appropriate. Will likely defer consolidating naming of `origin` vs `source` for blockchainID and senderAddress to another issue.